src/jit/emitarm64.cpp

```diff
@@ -7991,15 +8042,14 @@ BYTE*               emitter::emitOutputLJ(insGroup  *ig, BYTE *dst, instrDesc *i
     }
     else if (loadLabel)
     {
-        if (fmt == IF_LS_1A)       // LS_1A   XX......iiiiiiii iiiiiiiiiiittttt      Rt simm21
+        if (fmt == IF_LS_1A)       // LS_1A   XX...V..iiiiiiii iiiiiiiiiiittttt      Rt simm21
         {
-            // INS_ldr or INS_ldrsw (PC-Relative)
+            // INS_ldr (PC-Relative)
+            code |= insEncodeDatasizeVLS(code, id->idOpSize());  // XX V
```

Comment:
The method insEncodeDatasizeVLS will incorrect set  bit 30 for the ldrsw instruction